packages/vite/src/node/utils.ts

```diff
@@ -464,3 +465,30 @@ export function combineSourcemaps(
 export function unique<T>(arr: T[]): T[] {
   return Array.from(new Set(arr))
 }
+
+export function logHostInfo(
```

Comment:
How about put it into `./logger`? 😄

Reply:
Will do after `Apex: Legends` rounds :slightly_smiling_face:

Reply:
Do you also have suggestions for a better name?

Reply:
I personally would say `printServerUrls`